plutus-scb/src/Plutus/SCB/Events/Wallet.hs

```diff
@@ -11,4 +11,4 @@ import           Ledger       (Tx)
 newtype WalletEvent =
     BalancedTx Tx
     deriving (Show, Eq, Generic)
-    deriving anyclass (FromJSON, ToJSON)
+    deriving newtype (FromJSON, ToJSON)
```

Comment:
dangerous change, is this stuff tested anywhere?

Reply:
There is some basic encoding testing. I'm going to flesh it out, but not in this commit. 👍